common/utils/constantStrings.json

```diff
@@ -4,5 +4,6 @@
   "donateUrl": "https://www.freecodecamp.org/donate",
   "forumUrl": "https://forum.freecodecamp.org",
   "githubUrl": "https://github.com/freecodecamp/freecodecamp",
-  "RSA": "https://forum.freecodecamp.org/t/the-read-search-ask-methodology-for-getting-unstuck/137307"
+  "RSA": "https://forum.freecodecamp.org/t/the-read-search-ask-methodology-for-getting-unstuck/137307",
+  "homeURL": "https://www.freecodecamp.org/challenges/learn-how-freecodecamp-works"
```

Comment:
The home url is `https://freecodecamp.org`

Reply:
Alright I will update the link held in the variable, and that should update the pull request for approval.